src/app/zones/data/index.ts

```diff
@@ -12,25 +22,16 @@ export function getZoneDpServerAuthType(zone: ZoneOverview): string {
   }
   return ''
 }
+// TODO(jc): end
 
-export function getZoneControlPlaneStatus(zoneOverview: ZoneOverview): StatusKeyword | 'disabled' {
+export function getZoneControlPlaneStatus(zoneOverview: ZoneOverview): 'online' | 'offline' | 'disabled' {
+  // TODO(jc): should I check disabled for newly created zones for the purposes of this PR?
```

Comment:
Ah these ones are a real questions for this PR

- Is it ok to use the config from the first subscription we find here? (i'll address that outside this PR if its not ok)
- Should I check disabled for newly created zones for the purposes of this PR? (this one should be done here if necessary)

Reply:
From reading the review comments here, I think the answers are:

> Is it ok to use the config from the first subscription we find here? (i'll address that outside this PR if its not ok)

Yes

> Should I check disabled for newly created zones for the purposes of this PR? (this one should be done here if necessary)

Yes